src/Util.hpp

```diff
@@ -1,6 +1,6 @@
 // some utility functions
 
-#include "rack0.hpp" //TODO: ??? - stringf -> string::f
+#include "rack.hpp" //TODO: ??? - stringf -> string::f
```

Comment:
let's remove the `//TODO: ???` from here, we've changed `stringf` to `string::f` and there doesn't seem to be anything else to do when changing from rack0.hpp to rack.hpp.